sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_operations/_patch.py

```diff
@@ -46,10 +57,228 @@
 ClsType = Optional[Callable[[PipelineResponse[HttpRequest, HttpResponse], T, Dict[str, Any]], Any]]  # type: ignore
 
 
+ResponseType = Union[HttpResponse, AsyncHttpResponse]
+PipelineResponseType = PipelineResponse[HttpRequest, ResponseType]
+PollingReturnType_co = TypeVar("PollingReturnType_co", covariant=True)
+
+_FINISHED = frozenset(["succeeded", "cancelled", "cancelling", "failed"])
+_FAILED = frozenset(["validationfailed"])
+
+
+def convert_status(status, ll=False):
```

Comment:
Nit: was this method public in the old sdk? Wondering if it needs an `_` in the name?